frontend/src/forms/ParcelDetailForm.tsx

```diff
@@ -208,20 +206,19 @@ const ParcelDetailForm = (props: ParcelPropertyProps) => {
               )}
               <Row noGutters>
                 <Col>
-                  <h3>Address</h3>
+                  <h3>Parcel Information</h3>
                   <Form.Row className="pidPinForm">
                     <PidPinForm disabled={props.disabled} />
                     <AddressForm {...formikProps} disabled={props.disabled} nameSpace="address" />
                   </Form.Row>
-                  <Form.Row className="sumFinancialsForm">
-                    <SumFinancialsForm {...formikProps} />
-                  </Form.Row>
                 </Col>
               </Row>
               <Row noGutters>
                 <Col>
-                  <h3>Land</h3>
                   {<LandForm {...formikProps} disabled={props.disabled}></LandForm>}
+                  <Form.Row className="sumFinancialsForm">
+                    <SumFinancialsForm {...formikProps} />
+                  </Form.Row>
```

Comment:
Something to think about, no need to change. Do you know you could use the formik context in your subforms and not require the props to be passed down? That's how I implemented the common form elements if you want to take a look.
Unless there's some gotcha that I'm unaware.